.travis.yml

```diff
@@ -19,7 +19,7 @@ before_script:
   - bundle exec rake setup
 
 script:
-  - bundle exec rake test
+  - bundle exec rake test lint
```

Comment:
since there is default task 
https://github.com/activeadmin/activeadmin/blob/master/Rakefile#L48

why not 

```
 bundle exec rake 
```